projects/plugins/boost/app/assets/src/js/pages/settings/sections/Modules.svelte

```diff
@@ -141,4 +151,8 @@
 		transform: translateY( -4.5px );
 		display: inline-block;
 	}
+
+	[slot='notice'] {
```

Comment:
Wow, TIL. Is this a svelte trick?

Reply:
Nope, just a CSS attribute selector. Svelte keeps the slot attr on the rendered element.